### PR TITLE
config: make properties index distance configurable

### PR DESCRIPTION
**Pull request by Connor1996**
## What have you changed? (mandatory)
make range properties index distance configurable, that is, add two configs `prop_size_index_distance` and `prop_keys_index_distance` for each cf.

## What are the type of the changes? (mandatory)
- New feature (change which adds functionality)

## How has this PR been tested? (mandatory)
No need

## Does this PR affect documentation (docs) update? (mandatory)
No, these configs are only for test

## Does this PR affect tidb-ansible update? (mandatory)
No

### CODE REVIEW

File: src/raftstore/coprocessor/properties.rs [hunk 1]
```diff
@@ -383,7 +383,7 @@ impl TablePropertiesCollector for SizePropertiesCollector {
         self.index_handle.size += size;
         self.index_handle.offset += size as u64;
         // Add the start key for convenience.
-        if self.last_key.is_empty() || self.index_handle.size >= PROP_SIZE_INDEX_DISTANCE {
+        if self.last_key.is_empty() || self.index_handle.size >= DEFAULT_PROP_SIZE_INDEX_DISTANCE {
```
Comment (siddontang) [inline src/raftstore/coprocessor/properties.rs]: why we still use DEFAULT value here? 
should we use the configuration now?

Comment (Connor1996) [reply src/raftstore/coprocessor/properties.rs]: SizeProperties is only for compatibility, we use RangeProperties instead after v2.1

**Comment by Connor1996** [comment]
PTAL

**Pull request merged**